x-pack/plugins/lists/public/exceptions/hooks/persist_exception_item.test.ts

```diff
@@ -5,7 +5,8 @@
  * 2.0.
  */
 
```

Comment:
while running this test file locally, I seen multiple console errors:

```
  console.error
    Warning: It looks like you're using the wrong act() around your test interactions.
    Be sure to use the matching version of act() corresponding to your renderer:

    // for react-dom:
    import {act} from 'react-dom/test-utils';
    // ...
    act(() => ...);

    // for react-test-renderer:
    import TestRenderer from react-test-renderer';
    const {act} = TestRenderer;
    // ...
    act(() => ...);
        at TestComponent (/kibana/node_modules/@testing-library/react-hooks/lib/helpers/createTestHarness.js:22:5)
        at Suspense
        at ErrorBoundary (/kibana/node_modules/react-error-boundary/dist/react-error-boundary.cjs.js:40:35)
  ````
  
  There errors do not happen on main branch

Reply:
Same happens in other files in detection-engine, that test hooks

Reply:
@vitaliidm thanks for raising this, a fix has been included that resolves this